BonusCalcApi/V1/Factories/ResponseFactory.cs

```diff
@@ -47,6 +47,37 @@ public static OperativeSummaryResponse ToResponse(this OperativeSummary operativ
             };
         }
 
+        public static OutOfHoursSummaryResponse ToResponse(this OutOfHoursSummary outOfHoursSummary)
+        {
+            return new OutOfHoursSummaryResponse
+            {
+                Id = outOfHoursSummary.Id,
+                Name = outOfHoursSummary.Name,
+                Trade = new TradeResponse
+                {
+                    Id = outOfHoursSummary.TradeId,
+                    Description = outOfHoursSummary.TradeDescription
+                },
```

Comment:
This part has been used twice.

Reply:
@LBHCallumM not sure what you're suggesting? I'm constructing a `TradeResponse` object from properties of the `OvertimeSummary` and `OutOfHoursSummary` objects respectively - how would you suggest I go about it?